classes/task/insert_recordings.php

```diff
@@ -65,10 +62,15 @@ public function execute() {
                 $record->start_time = $rec->recording_start;
                 $record->end_time = $rec->recording_end;
                 $record->status = $rec->status;
-                $insert_record = $DB->insert_record('zoom_recordings', $record);
-                if(!empty($insert_record)) {
+                $zoom_recordings = $DB->insert_record('zoom_recordings', $record);
+                if(is_int($zoom_recordings)) {
                     $DB->update_record('event', (object)['id' => $value->id, 'recording_created' => 1]);
                 }
+            } else {
+                mtrace('No recordings found for the meeting_id: '. $value->meeting_id);
+            }
+            } catch (\moodle_exception $error) {
+                mtrace('No recordings found for the meeting_id: '. $value->meeting_id);
```

Comment:
@nayanalinkstreet should we not print the caught exception here?

Reply:
@Naveen1895 updated